pkg/appmanager/outputConfig.go

```diff
@@ -125,18 +138,8 @@ func (appMgr *Manager) outputConfigLocked() {
 		}
 	}
 
-	for _, profile := range appMgr.customProfiles.profs {
-		initPartitionData(resources, profile.Partition)
-		resources[profile.Partition].CustomProfiles = append(resources[profile.Partition].CustomProfiles, profile)
-	}
-	for _, irule := range appMgr.irulesMap {
-		initPartitionData(resources, irule.Partition)
-		resources[irule.Partition].IRules = append(resources[irule.Partition].IRules, *irule)
-	}
-	for _, intDg := range appMgr.intDgMap {
-		initPartitionData(resources, intDg.Partition)
-		resources[intDg.Partition].InternalDataGroups = append(resources[intDg.Partition].InternalDataGroups, *intDg)
-	}
+	// Copy everything from resources except CustomProfiles to be used for debug logging
```

Comment:
This should be inside the if block at line 160 so the copy is only done when in debug mode.